registry/quilt_server/views.py

```diff
@@ -1651,17 +1647,17 @@ def list_users_detailed():
     users = requests.get(user_list_api, headers=auth_headers).json()
 
```

Comment:
Suggestion: create a `usernames` variable and use it later:
```
usernames = (user['username'] for user in users['results'])
```

(It's a generator, so doesn't even build a list unnecessarily.)

Reply:
What would be the utility of this?